frigate/http.py

```diff
@@ -1954,9 +1955,62 @@ def export_recording(camera_name: str, start_time, end_time):
     )
 
 
+def export_filename_check_extension(filename: str):
+    filename_split = os.path.splitext(filename)
+    if filename_split[1] == ".mp4":
```

Comment:
This should check split[-1] to catch cases where . is in the name

Reply:
What do you exaclty mean Nick? Is it a problem when the filename contains a period character?

Reply:
Yes, since we know the file extension will come after the last . And we want to avoid issues where file name includes .

Reply:
In that case a simple `filename.endswith(".mp4")` and a change from `re.search(r'[^.:_A-Za-z0-9]'` to `re.search(r'[^:_A-Za-z0-9]'` will do as well?

Reply:
Yes, though I'm not 100% sure what the regex is checking for

Reply:
To make sure that we only allow alphanumeric characters, along with : and _ characters in the new filename.